src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -117,9 +117,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $encoderFactory = $this->encoderFactory ?: $this->getContainer()->get('security.encoder_factory');
         $encoder = $encoderFactory->getEncoder($userClass);
-        $bcryptWithoutEmptySalt = !$emptySalt && $encoder instanceof BCryptPasswordEncoder;
+        $saltlessWithoutEmptySalt = !$emptySalt && $encoder instanceof SelfSaltingEncoderInterface;
```

Comment:
`$encoder instanceof SelfSaltingEncoderInterface` is returning `false` when `$encoder` is an instance of [`Symfony\Component\Security\Core\Encoder\BCryptPasswordEncoder`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Security/Core/Encoder/BCryptPasswordEncoder.php) - anyone have any ideas why?
I'm probably missing something really obvious, but there's way too much brain fog at the moment for me.